README.md

```diff
@@ -14,6 +14,33 @@ And then execute:
 
 ## Usage
 
+### Automatic error reporting
+
+If you include `govuk_app_config` in your `Gemfile` your application will be automatically configured to send errors to Sentry.
```

Comment:
Added some more in https://github.com/alphagov/govuk_app_config/pull/1/commits/b3de5ab0bbd04e6cb9e3dcf2c81f69edf2a7d56a.